packages/normalizr/src/schemas/ImmutableUtils.ts

```diff
@@ -12,7 +12,6 @@
  */
 export function isImmutable(object: any) {
   return !!(
-    object &&
     typeof object.hasOwnProperty === 'function' &&
```

Comment:
removing this line, with a type of any/unknown, will throw when you try to access `object.hasOwnProperty` (because of null or undefined). if that's intentional, then the type should be changed to `{}` since that's the type that matches "object-coercible"